recipe/meta.yaml

```
@@ -38,10 +38,12 @@ requirements:
     - netcdf4
     - output_viewer >=1.3.0
     - numpy >=2.0.0,<3.0.0
+    - pyogrio
```

Added for fixing a pip check error: geopandas 1.1.1 requires pyogrio, which is not installed.
This might be a dependency newly introduced because of the inclusion of uxarray? Not sure if we need this in e3sm_diags repo.

---

This should ideally be reported to the geopandas maintainers on conda-forge and fixed there. This is ok as a temporary fix

---

This is not a problem on geopandas
https://github.com/conda-forge/geopandas-feedstock/blob/main/recipe%2Fmeta.yaml#L53

It must be a problem with the version reported by pyogrio itself.

---

Ah, I take that back. Probably a problem with geopandas-base not including this.

---

https://github.com/geopandas/geopandas/blob/v1.1.1/pyproject.toml

Yeah, this should be fixed on geopandas-base:

---

Reported in https://github.com/conda-forge/geopandas-feedstock/issues/125

---

```suggestion
    - pyogrio >= 0.7.2
```
See https://github.com/geopandas/geopandas/blob/v1.1.1/pyproject.toml#L25C6-L25C22

---

Trying to fix this in https://github.com/conda-forge/geopandas-feedstock/pull/126

---

@xylar, Thank you for helping with a proper fix!